src/Tools/ExternalAccess/Razor/RazorPredefinedCodeRefactoringProviderNames.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeRefactorings;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    internal static class RazorPredefinedCodeRefactoringProviderNames
```

Comment:
💡 Only include the ones Razor needs to use. Otherwise, any change to the internal name of _any_ refactoring means we have to coordinate with another project.

Reply:
> Otherwise, any change to the internal name of any refactoring means we have to coordinate with another project.

If the internal name change is for a refactoring not supported by razor, would we still need to co-ordinate?

I'd prefer to have all these accessible if possible to simplify the process of adding support for new razor C# code fix/refactoring providers. Without these in place we'd have to make Roslyn changes / package upgrades for every time we'd like to add a code fix/refactoring.